Formula/whistle.rb

```diff
@@ -3,9 +3,9 @@
 class Whistle < Formula
   desc "HTTP, HTTP2, HTTPS, Websocket debugging proxy"
   homepage "https://github.com/avwo/whistle"
-  url "https://registry.npmjs.org/whistle/-/whistle-2.4.2.tgz"
+  url "https://registry.npmjs.org/whistle/-/whistle-2.4.4.tgz"
   # version "2.4.2"
```

Comment:
Could you delete this comment, please?

Reply:
Sure! I'll make a pr later.

Reply:
done

Reply:
These trivial changes don't require a separate PR, let's add them into this PR. I would say let's add them to the commit with version bump.

Reply:
Got it. Thank you.